TST: Specviz2d remote data test fails

Did the data file change?

```
        # 3 new trace objects should have been loaded and plotted in the spectrum-2d-viewer
>       assert len(sp2dv.figure.marks) == 14
E       AssertionError: assert 15 == 14
E        +  where 15 = len([FRBImage(image=array([[[0, 0, 0, 0],\n        [0, 0, 0, 0],\n        [0, 0, 0, 0],\n        [0, 0, 0, 0],\n        [0, 0,...: 'vertical', 'dimension': 'y'}, 'color': {'dimension': 'color'}}, tooltip_style={'opacity': 0.9}, visible=False), ...])
E        +    where [FRBImage(image=array([[[0, 0, 0, 0],\n        [0, 0, 0, 0],\n        [0, 0, 0, 0],\n        [0, 0, 0, 0],\n        [0, 0,...: 'vertical', 'dimension': 'y'}, 'color': {'dimension': 'color'}}, tooltip_style={'opacity': 0.9}, visible=False), ...] = Figure(axes=[Axis(grid_lines='none', label='x: pixels', label_offset='40', scale=LinearScale(allow_padding=False, max=... padding_y=0.0, scale_x=LinearScale(allow_padding=False, max=-0.5, min=386.5), scale_y=LinearScale(max=43.5, min=-0.5)).marks
E        +      where Figure(axes=[Axis(grid_lines='none', label='x: pixels', label_offset='40', scale=LinearScale(allow_padding=False, max=... padding_y=0.0, scale_x=LinearScale(allow_padding=False, max=-0.5, min=386.5), scale_y=LinearScale(max=43.5, min=-0.5)) = <jdaviz.configs.mosviz.plugins.viewers.MosvizProfile2DView object at 0x7f1bf14d4d90>.figure

specviz2d/plugins/spectral_extraction/tests/test_spectral_extraction.py:67: AssertionError
```

--- a/jdaviz/configs/specviz2d/plugins/spectral_extraction/tests/test_spectral_extraction.py
+++ b/jdaviz/configs/specviz2d/plugins/spectral_extraction/tests/test_spectral_extraction.py
@@ -64,7 +64,7 @@ def test_plugin(specviz2d_helper):
     assert isinstance(trace, tracing.ArrayTrace)
 
     # 3 new trace objects should have been loaded and plotted in the spectrum-2d-viewer
-    assert len(sp2dv.figure.marks) == 14
+    assert len(sp2dv.figure.marks) == 15
 
     # interact with background section, check marks
     pext.trace_trace_selected = 'New Trace'

--- a/jdaviz/configs/specviz2d/plugins/spectral_extraction/tests/test_spectral_extraction.py
+++ b/jdaviz/configs/specviz2d/plugins/spectral_extraction/tests/test_spectral_extraction.py
@@ -63,6 +63,8 @@ def test_plugin(specviz2d_helper):
     trace = pext.export_trace(add_data=True)  # overwrite
     assert isinstance(trace, tracing.ArrayTrace)
 
+    # TODO: Investigate extra hidden mark from glue-jupyter, see
+    # https://github.com/spacetelescope/jdaviz/pull/2478#issuecomment-1731864411
     # 3 new trace objects should have been loaded and plotted in the spectrum-2d-viewer
     assert len(sp2dv.figure.marks) == 15
 

